ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/validator/ValidationFailed.scala

```diff
@@ -3,18 +3,12 @@
 
 package com.daml.ledger.validator
 
-sealed trait ValidationResult
+sealed trait ValidationFailed extends RuntimeException
```

Comment:
Would it make sense to extends `NoStackTrace` here?

Reply:
Yeah, probably; we always handle it and honestly, I think we're going to get rid of it eventually.